### PR TITLE
mgr: optimization some judgment and adjust the debug remove value in register_new_pgs

### PR DESCRIPTION
**Pull request by songbaisen**
mgr: optimization some judgment and adjust the debug remove value in register_new_pgs

Signed-off-by: song baisen <song.baisen@zte.com.cn>

### CODE REVIEW

**Comment by songbaisen** [comment]
jenkins test this please

File: src/mgr/ClusterState.cc [hunk 1]
```diff
@@ -76,16 +76,13 @@ void ClusterState::ingest_pgstats(MPGStats *stats)
...
-    if (pg_map.pg_stat.count(pgid) &&
-        pg_map.pg_stat[pgid].get_version_pair() > pg_stats.get_version_pair()) {
+     // In case we already heard about more recent stats from this PG
+     // from another OSD
+    } else if (pg_map.pg_stat[pgid].get_version_pair() > pg_stats.get_version_pair()) {
       dout(15) << " had " << pgid << " from " << pg_map.pg_stat[pgid].reported_epoch << ":"
                << pg_map.pg_stat[pgid].reported_seq << dendl;
       continue;
-    }
+    }
```
Comment (jcsp) [inline src/mgr/ClusterState.cc]: Please remove this accidental whitespace at the end of the line

**Comment by jcsp** [comment]
This change looks sensible to me -- please could you just remove that little bit of whitespace, and also edit the commit message to start "mgr: "

**Comment by songbaisen** [comment]
@jcsp  Thank you jscp. Done it.

**Pull request merged**